### ISSUE TITLE
Ridge polynomials (on the disk)?

### ISSUE THREAD

**Issue by MikaelSlevinsky** [JuliaApproximation/MultivariateOrthogonalPolynomials.jl#89]
These look like `P_{n,k}(x,y) = U_n(x * cos(k*pi/(n+1)) + y * sin(k*pi/(n+1)))`, orthogonal on the unweighted unit disk.

Sparse differentiation is trivial (using ultraspherical generalizations), but the Jacobi operators are not obvious. Letting `z = x * cos(k*pi/(n+1)) + y * sin(k*pi/(n+1))`, then multiplication by this linear combination is tridiagonal-block-tridiagonal. Maybe there's another combination from which the Jacobi operators can be untangled?

**Comment by MikaelSlevinsky** [comment]
https://www.math.auckland.ac.nz/~waldron/Preprints/Disc-Polys/discpolys.pdf

**Comment by dlfivefifty** [comment]
Yes I know this family from Dunkl and Xu, it’s cool and doesn’t require any high order OPs
